python_modules/dagstermill/dagstermill/__init__.py

```diff
@@ -60,6 +65,9 @@
 class DagstermillInNotebookExecutionContext(AbstractTransformExecutionContext):
     def __init__(self, pipeline_context):
         self._pipeline_context = pipeline_context
```

Comment:
if you add a check.inst_param up here I believe you will catch the bug I mentioned at 158